src/pages/rekey-reason/rekey-reason.effects.ts

```diff
@@ -34,4 +39,20 @@ export class OfficeEffects {
       ];
     }),
   );
+
+  @Effect()
+  findUserEffect$ = this.actions$.pipe(
+    ofType(rekeyActions.REKEY_REASON_FIND_USER),
+    switchMap((action: rekeyActions.RekeyReasonFindUser) => {
+      return this.findUserProvider.userExists(action.staffNumber)
+        .pipe(
+          map((response: any) => {
```

Comment:
does the type `HttpResponse` not work here?

Reply:
Works fine `HttpResponse<any>`